src/components/Card.js

```diff
@@ -12,8 +12,8 @@ export const Card = (props) => {
   } = props;
 
   return (
-    <div className="card" style={{ backgroundImage: `url(${backgroundImage})` }}>
-      <a href={link}>
+    <div className="card" style={{ backgroundImage: 'url(https://bit.ly/2B4NZ51)' }}>
+      <NavLink to={`/articles/${link}`}>
```

Comment:
Should we be using the slug to fetch a single article or id? If yes, hope it won't amount to extra work going forward?

Reply:
This link here does not infer what we're using to fetch articles from the backend. I believe we are using id not slug.